articles/iot-hub/iot-hub-devguide-direct-methods.md

```diff
@@ -167,7 +167,7 @@ The device sends responses to `$iothub/methods/res/{status}/?$rid={request id}`,
 
 * The `status` property is the device-supplied status of method execution.
 
-* The `$rid` property is the request ID from the method invocation received from IoT Hub.
+* The `$rid` property is the request ID from the method invocation received from IoT Hub and request ID contain value in hexadecimal formate.
```

Comment:
@SumitBalia Please make this editorial fix, and then I can approve this pull request. Thank you.

```suggestion
* The `$rid` property is the request ID from the method invocation received from IoT Hub. The request ID is a hexadecimal formatted value.
```

Reply:
Thank you @kgremban for your suggestion. Done the changes based on your suggestion.